lib/logging/activity-event.js

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const logger = require('./')('activity');
+
+module.exports = function activityEvent(event, uid, client_id, userAgent) {
```

Comment:
I suggest taking the full `request` object as argument here and pulling the user-agent out of it, in case we decide to log more details in future.  This more closely parallels what is done in the auth-server and it can still be `null` for server-to-server requests.